test/test_format.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I think this file is just an accidental copy of `test_lint.sh`. It seems have the same contents and doesn't seem to be used anywhere. Please remove.